frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -292,15 +298,21 @@ const ProjectListView: React.FC<IProps> = ({ filterable, title, mode }) => {
           />
         )}
         <Container fluid className="TableToolbar">
-          <h3 className="mr-auto">{title}</h3>
-          <Button className="mr-2" onClick={() => fetch('excel', 'generic')}>
-            Export Generic Report
-          </Button>
-          <Button className="mr-2" onClick={() => fetch('csv', 'generic')}>
-            Export CSV
-          </Button>
-          {keycloak.hasClaim(Claims.ADMIN_PROJECTS) && (
-            <Button onClick={() => fetch('excel', 'spl')}>Export SPL Report</Button>
+          <h3 className="mr-4">{title}</h3>
```

Comment:
Updates to permissions and display on the project list view. also using icons instead of text buttons.